fastserde/avro-fastserde-tests-common/src/test/avro/logicalTypesDefined.avsc

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "record",
+  "name": "FastSerdeLogicalTypesDefined",
+  "namespace": "com.linkedin.avro.fastserde.generated.avro",
+  "doc": "Used in tests to confirm fast-serde supports logical-types",
+  "fields": [
+    {
+      "name": "timeMillisField",
+      "type": {
+        "type": "int",
+        "logicalType": "time-millis"
+      }
+    },
+    {
+      "name": "dateField",
+      "type": {
+        "type": "int",
+        "logicalType": "date"
+      }
+    },
+    {
+      "name": "arrayOfUnionOfDateAndTimestampMillis",
+      "type": {
+        "type": "array",
+        "items": [
+          {
+            "type": "int",
+            "logicalType": "date"
+          },
+          {
+            "type": "long",
+            "logicalType": "timestamp-millis"
+          }
+        ]
+      }
+    }
```

Comment:
Do we also aim to support custom logical types? Can we provide a logical type, test case & conversions for such a scenario?

Reply:
Yes, I'll do that using `fixed` raw-type converted to/from `Duration` logical type (coming soon).